certified-connectors/Calendly/apiDefinition.swagger.json

```diff
@@ -371,7 +371,8 @@
                     "default": "invitee.created",
                     "enum": [
                       "invitee.created",
-                      "invitee.canceled"
+                      "invitee.canceled",
```

Comment:
Can you please correct canceled spelling to Cancelled?

Reply:
We cannot change this event name as it would be a breaking change in our API specification:
[https://developer.calendly.com/api-docs/c1ddc06ce1f1b-create-webhook-subscription](https://developer.calendly.com/api-docs/c1ddc06ce1f1b-create-webhook-subscription)